jenkins.sh

```diff
@@ -13,6 +13,10 @@ docker ps -a | grep 'Exited' |  awk ' { print $1; } ' | xargs docker rm || true
 # done manually, as needed.
 
 failed=0
+./build-image.sh arch                          || failed=1
+./build-image.sh arch-gcc                      || failed=1
+./build-image.sh arch-clang                    || failed=1
```

Comment:
These images seem to be missing?

Reply:
I'm not a member of rhub, so I don't believe I will be able to push the images. If you want to add me, I'd be happy to do it. Otherwise I think you will likely need to push the images.

Reply:
If you want, I can also remove the `jenkins.sh` entries for now and we can add them back once the images have been pushed. Just let me know..

Reply:
What I am asking is that you need to want three images: `rhub/arch`, `rhub/arch-gcc` and `arch-clang`? No? Where are the Dockerfiles for those? The arch image does not have R installed.

Reply:
Ah apologies; I forgot to include them in the commit. I'll fix that now. Also, I'm not entirely sure that three images are necessary, but rather I'm just following the convention used for the Debian / Fedora images, with the assumption that some people will prefer one or the other.

Reply:
Thanks! I am still missing something. Seems like your `arch` image already has R. What are the `arch-gcc` and `arch-clang` images for, then? 

Note that for e.g. Debian we have sg. like
* `debian`: base image of all R-hub Debian images
* `debian-gcc`: add GCC
* `debign-gcc-devel`: add R-devel

But seems like you just want a single image with R installed from the default Arch Linux source? Which is completely fine, but then we do not need the `arch-clang` and `arch-gcc` images. R will use the same compiler as the one that was used to compile it, anyway. So it is enough to make that available on the image.

Reply:
Ah got it. I started with a single image, but then thought I would mirror the setup for Fedora/Debian, in case it was needed for some purpose. I'll revert to a single image in that case.